package.json

```diff
@@ -262,6 +264,10 @@
     }
   },
   "lint-staged": {
+    "public/locales/**/*.{json,po}": [
+      "npm run convert-translations",
+      "git add public/locales/**/*.json public/locales/**/*.po"
```

Comment:
This `git add` might not be required since `lint-staged` already does it for us — unless the specific path requires it?

Reply:
I agree with this observation, I also hoped the converted files would get staged automatically. However, they are left behind without this.

Reply:
We removed the related lint-staged config in 0b8f8609efc765854a48b60d1e5d76bdaddabef0 as suggested in [a review](https://github.com/Trustroots/trustroots/pull/1289#discussion_r396008019).